vmm/src/config.rs

```diff
@@ -397,16 +406,59 @@ impl FromStr for ByteSized {
     }
 }
 
+pub enum CpuTopologyParseParseError {
```

Comment:
Is it a typo that you named this enum `CpuTopologyParseParseError` instead of `CpuTopologyParseError`?